pkg/plugin/linuxutil/Makefile

```diff
@@ -5,7 +5,7 @@ MOCKGEN = $(TOOLS_BIN_DIR)/mockgen
 .PHONY: generate
 
 generate: $(MOCKGEN) ## Generate mock clients
-	$(MOCKGEN) -source=$(REPO_ROOT)/pkg/plugin/linuxutil/types_linux.go -copyright_file=$(REPO_ROOT)/pkg/lib/ignore_headers.txt -package=linuxutil > linuxutil_mock_generated.go
+	$(MOCKGEN) -source=$(REPO_ROOT)/pkg/plugin/linuxutil/types_linux.go -copyright_file=$(REPO_ROOT)/pkg/lib/ignore_headers.txt -package=linuxutil > linuxutil_mock_generated_linux.go
```

Comment:
Fix for build failures on Windows ARM64:
```
  Running [/home/runner/golangci-lint-1.62.2-linux-amd64/golangci-lint run --new-from-patch=/tmp/tmp-1820-ogekMZGZuwPN/pull.patch --new=false --new-from-rev= --concurrency 4 --verbose --config=.golangci.yaml --timeout=25m] in [/home/runner/work/retina/retina] ...
  Error: pkg/plugin/linuxutil/linuxutil_mock_generated.go:15:10: could not import github.com/cakturk/go-netstat/netstat (-: # github.com/cakturk/go-netstat/netstat
  Error: ../../../go/pkg/mod/github.com/cakturk/go-netstat@v0.0.0-20200220111822-e5b49efee7a5/netstat/netstat.go:42:9: undefined: skStates
  Error: ../../../go/pkg/mod/github.com/cakturk/go-netstat@v0.0.0-20200220111822-e5b49efee7a5/netstat/netstat.go:55:9: undefined: osTCPSocks
  Error: ../../../go/pkg/mod/github.com/cakturk/go-netstat@v0.0.0-20200220111822-e5b49efee7a5/netstat/netstat.go:61:9: undefined: osTCP6Socks
  Error: ../../../go/pkg/mod/github.com/cakturk/go-netstat@v0.0.0-20200220111822-e5b49efee7a5/netstat/netstat.go:67:9: undefined: osUDPSocks
  Error: ../../../go/pkg/mod/github.com/cakturk/go-netstat@v0.0.0-20200220111822-e5b49efee7a5/netstat/netstat.go:73:9: undefined: osUDP6Socks) (typecheck)
  	netstat "github.com/cakturk/go-netstat/netstat"
  	        ^
```

Reproducible with `GOOS=windows GOARCH=arm64 go build ./...` but for some reason the previous version of golangci-lint ignored it

Reply:
i don't think the upstream library supports arm64 windows 🤔 also maybe older golangci-lint didn't run for windows arm64 by default?

Reply:
it's definitely in the GH action matrix: https://github.com/microsoft/retina/blob/a76f1b46249440b1ee50bc7d023b63dce5144624/.github/workflows/golangci-lint.yaml#L14-L15